### PR TITLE
Add `model_runs` directory (3rd attempt)

### PR DESCRIPTION
**Pull request by damonbayer**
Most of these changes were approved in https://github.com/CDCgov/pyrenew-hew/pull/136. Additional changes begin with https://github.com/CDCgov/pyrenew-hew/commit/26cf4aeca1bc715891c306579edbc5aee4c88675

### CODE REVIEW

File: pipelines/collate_plots.py [hunk 1]
```diff
@@ -136,17 +153,24 @@ def process_dir(
         named subdirectories. If ``None``, look in all
         subdirectories of ``base_dir``. Default ``None``.
     """
+    if save_dir is None:
+        save_dir = Path(base_dir).parent / "figures"
```
Comment (dylanhmorris) [inline pipelines/collate_plots.py:157]: Create if not exists?

File: pipelines/collate_plots.py [hunk 2]
```diff
@@ -167,10 +190,17 @@ def collate_from_all_subdirs(
...
+        Directory in which to save the merged PDFs.
+        If ``None``, use a "figures" directory in the parent directory of ``model_base_dir``. Default ``None``.
+
     Returns
     -------
     None
     """
+    if save_dir is None:
+        save_dir = Path(model_base_dir).parent / "figures"
+
```
Comment (dylanhmorris) [inline pipelines/collate_plots.py:203]: As above, create if not exists?

**Review by dylanhmorris — APPROVED** [review]
One minor suggestion, not critical

File: pipelines/collate_plots.py (hunk 1 above)
Comment (damonbayer) [reply pipelines/collate_plots.py:157]: This function calls `merge_pdfs_from_subdirs`, which makes the directory if it doesn't exist.

File: pipelines/collate_plots.py (hunk 2 above)
Comment (damonbayer) [reply pipelines/collate_plots.py:203]: This function eventually calls `merge_pdfs_from_subdirs`, which makes the directory.

**Pull request merged**